### ISSUE TITLE
"Enable Overscan" Toggle

### ISSUE THREAD

**Issue by SlyCooperReloadCoded** [xemu-project/xemu#1262]
### Feature Request

Some games such as SSX Tricky by default render additional pixels outside of the normal screen area, resulting in you being able to see things that are outside of the normal view, such as the edge of terrain culling or the outside edge of HUD elements.  Having overscan off by default but adding an "Enable Overscan" option in the Video settings would be a nice fix for this.

### Additional Context

Screenshot of SSX Tricky, showing it renders past the HUD's intended limits because of overscan:
![Capture](https://user-images.githubusercontent.com/58268914/190277171-2add477f-c9b7-48be-9e5c-86b4d4b46c13.PNG)

Additionally, when you warp in the game, the screen turns white, but overscan ruins this effect:
![Capture2](https://user-images.githubusercontent.com/58268914/190277255-007b2db8-6ad5-4340-8a54-86f18aa5c1f2.PNG)

**Comment by RinMaru** [comment]
yea i see this happen in allot of games and was going to report it but wasnt sure this was the issue.

**Comment by Bobble80** [comment]
@abaire 
Yes, this is the issue I’m seeing in a lot of games I’ve tested in Xemu lately. 
It would be great if an overscan option could be added to Xemu. 
I’m glad to see it’s not an actual bug, just needs an overscan toggle to fix.
